thirdparty/sokol/sokol_v.post.h

```diff
@@ -1,4 +1,4 @@
-#if defined(SOKOL_GLCORE33) || defined(SOKOL_GLES2) || defined(SOKOL_GLES3)
+#if defined(SOKOL_GLCORE33) || defined(SOKOL_GLES3)
 	void v_sapp_gl_read_rgba_pixels(int x, int y, int width, int height, unsigned char* pixels) {
 		glReadPixels(x, y, width, height, GL_RGBA, GL_UNSIGNED_BYTE, pixels);
 	}
```

Comment:
hm, that is interesting ... may be on windows all of them are not defined (the default may be some other backend now) ...

Reply:
According to <https://learn.microsoft.com/en-us/windows/win32/opengl/glreadpixels> , that function should be available after including Gl.h and linking to Opengl32.lib .

Reply:
perhaps the older Sokol did it, but the newer one does not 🤔

Reply:
ok, I think I know what is going on, I'll push a fix asap.

Reply:
I contributed it, but it got removed after a while I suspect it's because floooh don't use it anywhere so it's easy to forget in a rough "spring cleaning" I assume

Reply:
I have opened #19527 that hopefully fixes a lot of stuff